### PR TITLE
Add `/nick` to miniirc.py and README.md and update LICENSE.md

### PR DESCRIPTION
**Pull request by piecubed**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 The MIT License (MIT)
 
-Copyright © 2018 by luk3yx
+Copyright © 2018 by luk3yx and other developers of miniirc
```
Comment (luk3yx) [inline LICENSE.md]: This should link to [the contributors page](https://github.com/luk3yx/miniirc/graphs/contributors).

File: README.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ irc = miniirc.IRC(ip, port, nick, channels = None, *, ssl = None, ident = None,
 | `msg(target, *msg)`         | Sends a `PRIVMSG` to `target`.              |
 | `notice(target, *msg)`      | Sends a `NOTICE` to `target`.               |
 | `quote(*msg, force=None)` | Sends a raw message to IRC, use `force=True` to send while disconnected. Do not send multiple commands in one `irc.quote()`, as the newlines will be stripped and it will be sent as one command. |
+| `nick(new_nick)`            | Sends a `/nick` to irc server |
```
Comment (luk3yx) [inline README.md]: This list is supposed to be in alphabetical order, and `/nick` should be `NICK`.

`irc.nick` contains the (current) nickname of the IRC client, so this function will be removed from all IRC objects unless it has a different name.

File: miniirc.py [hunk 3]
```diff
@@ -146,6 +146,9 @@ def ctcp(self, target, *msg, reply = False):
 
     def me(self, target, *msg):
         return self.ctcp(target, 'ACTION', *msg)
+    
+    def nick(new_nick):
```
Comment (luk3yx) [inline miniirc.py]: `irc.nick` is already used for the nick of the IRC client, and will override this function.

This function will need to have an extra `self` parameter as it is in a class.

**Comment by luk3yx** [comment]
*Also, can you try to modify existing PRs instead of adding new ones? If you commit to the PR's branch, it will automatically add them to the existing PR.*

**Comment by piecubed** [comment]
***AHA!***

File: README.md [hunk 4]
```diff
@@ -34,6 +34,7 @@ irc = miniirc.IRC(ip, port, nick, channels = None, *, ssl = None, ident = None,
 
 | Function      | Description                                               |
 | ------------- | --------------------------------------------------------  |
+| `change_nick(new_nick)`            | Sends a `NICK` to irc server |
```
Comment (luk3yx) [inline README.md]: IRC is normally in capital letters.

Comment (luk3yx) [reply README.md]: "Sends a `NICK` to the IRC server." would probably be better.

File: LICENSE.md (hunk 1 above)
Comment (luk3yx) [reply LICENSE.md]: Fixed

File: miniirc.py [hunk 5]
```diff
@@ -147,7 +147,7 @@ def ctcp(self, target, *msg, reply = False):
     def me(self, target, *msg):
         return self.ctcp(target, 'ACTION', *msg)
     
-    def nick(new_nick):
+    def change_nick(new_nick):
```
Comment (luk3yx) [inline miniirc.py]: `TypeError: change_nick() takes 1 positional argument but 2 were given`

**Comment by piecubed** [comment]
made changes.

**Comment by piecubed** [comment]
done.

File: README.md [hunk 6]
```diff
@@ -42,7 +43,7 @@ irc = miniirc.IRC(ip, port, nick, channels = None, *, ssl = None, ident = None,
 | `Hander(...)` | An event handler, see [Handlers](#handlers) for more info.|
 | `main()`      | Starts the main loop in a thread if not already running.  |
 | `me(target, *msg)`          | Sends a `/me` (`CTCP ACTION`) to `target`.  |
-| `msg(target, *msg)`         | Sends a `PRIVMSG` to `target`.              |
+| `msg(target, *msg)` | Sends a `PRIVMSG` to `target`.              |
```
Comment (luk3yx) [inline README.md]: Why was this changed? It was in line with `me()` and `notice()`.

**Comment by luk3yx** [comment]
Closing for inactivity.
